views/components/form-fields-next/macros.njk

```diff
@@ -9,9 +9,7 @@
                 <ol class="form-errors__list">
                     {% for error in errors %}
                         <li>
-                            {% if error.field %}
-                                {{ error.field.label | safe }} &ndash;
-                            {% endif %}
+                            {% if error.label %}{{ error.label | safe }} &mdash;{% endif %}
```

Comment:
Made a change here, and in associated errors, to define a `label` here rather than passing the whole field @mattandrews